lib/api/apiUtils/object/versioning.js

```diff
@@ -129,10 +166,14 @@ export function versioningPreprocessing(bucketName, bucketMD, objectKey,
             if (nullVersionId === undefined) {
```

Comment:
when suspended, do you only delete the master version if it is a null version?  please update comment above